src/caselawclient/models/documents/__init__.py

```diff
@@ -329,31 +329,39 @@ def status(self) -> str:
 
         return DOCUMENT_STATUS_NEW
 
-    def force_enrich(self) -> None:
+    def force_enrich(self) -> bool:
```

Comment:
This is technically an API change, but nothing should care about something that doesn't return a value returning something.

Reply:
yeah, technically it is a breaking change, so just remember to reflect in the version bump